### DISCUSSION TITLE
请问address是否对二级域名不生效

### DISCUSSION THREAD

**Discussion by acp120** [pymumu/smartdns#1383] [General]
比如指定 address music.apple.com /1.2.3.4 但是这个不生效？请问如何指定 感谢
address apple.com / 1.2.3.4 就生效

**Comment by PikuZheng** [comment]
address /apple.com/1.2.3.4 包含 music.apple.com

**Reply by acp120** [reply]
如果只想指定 music.apple.com这个域名呢 毕竟指定全部的apple.com也不合适
